src/client/features/interactions/index.js

```diff
@@ -0,0 +1,135 @@
+const React = require('react');
+const h = require('react-hyperscript');
+const _ = require('lodash');
+const queryString = require('query-string');
+const Loader = require('react-loader');
+
+const make_cytoscape = require('../../common/cy/');
+
+const { ServerAPI } = require('../../services/');
+
+const { BaseNetworkView } = require('../../common/components');
+const { getLayoutConfig } = require('../../common/cy/layout');
+
+class Interactions extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      cy: make_cytoscape({ headless: true }),
+      componentConfig: {},
+      layoutConfig: {},
+      networkJSON: {},
+      networkMetadata: {
+        name: '',
+        datasource: '',
+        comments: []
+      },
+      loading: true,
+    };    
+    const query = queryString.parse(props.location.search);
+    ServerAPI.getNeighborhood(query.ID).then(res=>{ 
+      const layoutConfig = getLayoutConfig('interactions');
+      const componentConfig = _.merge({}, BaseNetworkView.config, { useSearchBar: true});
+      const network= this.parse(res,query.ID);
+      this.setState({
+        componentConfig: componentConfig,
+        layoutConfig: layoutConfig,
+        networkJSON: network ,
+        networkMetadata: {
+          name: query.ID,
+          datasource: 'Pathway Commons',
+          comments: ['place holder'], dataSource: ['place holder'], organism: ['place holder'], title: ['place holder'], uri: 'place holder'
+        },
+        loading: false
+      }); 
+    });
+  }
+
+  edgeTypeing(type){
+    switch(type){
+      case 'in-complex-with':
+      case 'interacts-with':
+        return 'Binding';
+      case 'controls-phosphorylation-of':
+        return 'Phosphorylation';
+      case 'controls-expression-of':
+        return 'Expression';
+      default:
+        return '';
+    }
+  }
+
+  findId(data,id){
+    return data.filter(line => line.includes(id))[0].split('	')[0];
+  }
+
+  pathwayLinks(line){
+    return line[6].split(';').map( link => {
+      const splitLink=link.split('/').reverse();
+      return [splitLink[1]==='reactome'? 'reactome': 'Pathway Commons',splitLink[0]];
+    });
+  }
+
+  addInteraction(splitLine,network,nodeMap){
```

Comment:
`parse()` should be handling all parsing.  Functions like `addInteraction()` should receive the already parsed data.

E.g. `splitLine` should be replaced by something like `nodeAId, nodeBId`